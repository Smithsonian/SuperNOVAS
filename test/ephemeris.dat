
2450545.000000 SUN        S0 O0 A1:    -0.006750     0.004995     0.002339    -0.000006    -0.000006    -0.000002     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2450545.000000 SUN        S0 O1 A1:    -0.006750     0.004995     0.002339    -0.000006    -0.000006    -0.000002     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2450545.000000 SUN        S0 O2 A1:    -0.006750     0.004995     0.002339    -0.000006    -0.000006    -0.000002     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2451545.000000 SUN        S0 O0 A1:    -0.007138    -0.002644    -0.000921     0.000005    -0.000007    -0.000003     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2451545.000000 SUN        S0 O1 A1:    -0.007138    -0.002644    -0.000921     0.000005    -0.000007    -0.000003     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2451545.000000 SUN        S0 O2 A1:    -0.007138    -0.002644    -0.000921     0.000005    -0.000007    -0.000003     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2452545.000000 SUN        S0 O0 A1:     0.000657    -0.004704    -0.002013     0.000008     0.000003     0.000001     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2452545.000000 SUN        S0 O1 A1:     0.000657    -0.004704    -0.002013     0.000008     0.000003     0.000001     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2452545.000000 SUN        S0 O2 A1:     0.000657    -0.004704    -0.002013     0.000008     0.000003     0.000001     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 
