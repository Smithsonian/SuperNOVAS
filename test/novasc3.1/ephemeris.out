  -10000.000 A0: ERROR 13 
  -10000.000 A0: ERROR 13 
  -10000.000 A0: ERROR 13 
  -10000.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.010 A0: ERROR 13 
   10000.010 A0: ERROR 13 
   10000.010 A0: ERROR 13 
   10000.010 A0: ERROR 13 
  -10000.000 A1: SUN        3    -0.000009     0.003773     0.001651    -0.009747    -0.002548    -0.000896 

  -10000.000 A1: SUN        3     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

  -10000.000 A1: EARTH      3     0.810058    -0.553762    -0.240106    17.374036    21.766877     9.439017 

  -10000.000 A1: EARTH      3     0.810068    -0.557535    -0.241757    17.383783    21.769425     9.439914 

       0.000 A1: SUN        3    -0.007138    -0.002644    -0.000921     0.009234    -0.011779    -0.005287 

       0.000 A1: SUN        3     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

       0.000 A1: EARTH      3    -0.184271     0.884782     0.383824   -29.784950    -5.028997    -2.180527 

       0.000 A1: EARTH      3    -0.177134     0.887425     0.384746   -29.794184    -5.017218    -2.175240 

   10000.000 A1: SUN        3    -0.000010    -0.004018    -0.001669     0.007845     0.006630     0.002675 

   10000.000 A1: SUN        3     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

   10000.000 A1: EARTH      3    -0.537235    -0.790493    -0.342586    24.755316   -14.602263    -6.330115 

   10000.000 A1: EARTH      3    -0.537226    -0.786475    -0.340917    24.747471   -14.608892    -6.332790 

   10000.000 A1: SUN        3    -0.000010    -0.004018    -0.001669     0.007845     0.006630     0.002675 

   10000.000 A1: SUN        3     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

   10000.000 A1: EARTH      3    -0.537235    -0.790493    -0.342586    24.755316   -14.602263    -6.330115 

   10000.000 A1: EARTH      3    -0.537226    -0.786475    -0.340917    24.747471   -14.608892    -6.332790 

   10000.010 A1: SUN        3    -0.000010    -0.004018    -0.001669     0.007844     0.006630     0.002675 

   10000.010 A1: SUN        3     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

   10000.010 A1: EARTH      3    -0.537093    -0.790577    -0.342623    24.757953   -14.598388    -6.328435 

   10000.010 A1: EARTH      3    -0.537083    -0.786560    -0.340954    24.750109   -14.605018    -6.331111 

